native/dpotf2.go

```diff
@@ -0,0 +1,66 @@
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dpotrf computes the cholesky decomposition of the symmetric positive definite
+// matrix a. If ul == blas.Upper, then a is stored as an upper-triangular matrix,
+// and a = U^T U is stored in place into a. If ul == blas.Lower, then a = L L^T
+// is computed and stored in-place into a. This is the unblocked version of the
+// algorithm. Dpotrf returns 0 if the decomposition is successful, otherwise
```

Comment:
Sorry to be difficult on this, but is the indormation we are discarding here likely to be useful to someone?

Making it an error allows that to continue to be used if needed.